packages/amplify-e2e-core/src/utils/auth-utils.ts

```diff
@@ -42,12 +42,8 @@ export async function setupUser(
   }
 }
 
-export async function addUserToGroup(
-  cognitoClient: CognitoIdentityServiceProvider,
-  userPoolId: string,
-  username: string,
-  groupName?: string,
-): Promise<void> {
+export async function addUserToGroup(userPoolId: string, username: string, groupName: string, region?: string): Promise<void> {
```

Comment:
Where do we use `addUserToGroup` in the repo? I somehow couldn't find it 😂

Reply:
In new test. I just changed signature to be easier to use since nothing else uses it.